File: wasm/bmxt-core/src/cmd/exit.rs
use crate::meta::Cmd;
use crate::model::{DispatchJson, Effect};

pub const CMD: Cmd = Cmd {
    name: "exit",
    aliases: &[],
    usage_primary: "exit",
};

pub fn run(_args: &[String]) -> DispatchJson {
    DispatchJson::effects(vec![Effect::ExitPane])
}

File: lib/features/bmxt-core/cmd/exit.ts
import type { CmdMeta } from "../types"
import { effectsDispatch } from "../types"

export const CMD: CmdMeta = {
  name: "exit",
  aliases: [],
  usagePrimary: "exit"
}

export function run(_args: string[]) {
  return effectsDispatch([{ kind: "exit_pane" }])
}
